decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -53,7 +53,12 @@ def external_icon(path)
       # Ugly hack to prevent PhantomJS from freaking out with SVGs.
       return content_tag(:span, "?", class: "external-svg", "data-src" => path) if Rails.env.test?
 
-      image_tag(path, class: "external-svg", style: "display: none")
+      if path.split(".").last == "svg"
+        asset = Rails.application.assets_manifest.find_sources(path).first
+        asset.gsub("<svg", '<svg class="icon"').html_safe
```

Comment:
I think it was @Dor3nz who modified the `icon` method in this same file to be able to customize what classes are being added to the svg. Can you try to use that same method to render this, or at least add this class customization capability?